lib/twilioconnection.js

```diff
@@ -30,14 +30,16 @@ const states = {
 
 const DEFAULT_MAX_CONSECUTIVE_MISSED_HEARTBEATS = 3;
 const DEFAULT_MAX_REQUESTED_HEARTBEAT_TIMEOUT = 5000;
+const DEFAULT_OPEN_TIMEOUT = 15000;
 const DEFAULT_WELCOME_TIMEOUT = 5000;
```

Comment:
Welcomes should be allowed for up to 15 seconds, and the hello should be sent up to 3 times at [0, 5, 10] seconds. It seems like this change should be part of this PR rather than https://github.com/twilio/twilio-video.js/pull/985 (JSDK-2777).

Reply:
Looks like #985 already covered that.

Reply:
looks like #985 already handled this requirement.